examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp

```diff
@@ -216,7 +216,7 @@ void drawTopology()
     wclear(topoPad);
     wattroff(topoPad, COLOR_PAIR(1));
     mvprintw(17, 10, "Mesh Topology");
-    mvwprintw(topoPad, nodeY > 15 ? nodeY - 16 : 0, 0, "");
+    mvwaddch(topoPad, nodeY > 15 ? nodeY - 16 : 0, 0, 0);
```

Comment:
This change is my only concern. I want to be sure its behavior is as expected. Have you actually run this modified example?

Reply:
Nope, I assumed you tested it hahaha

Reply:
Just did a quick run and it looks the same. I think we're good